index.html

```diff
@@ -1,23 +1,78 @@
 <!DOCTYPE html>
 <html>
-	<body>
-		<h1>My First Heading</h1>
-		<p>My first paragraph.</p>
-
-		<div id="target"></div>
-
-		<script src="https://cdn.jsdelivr.net/npm/ractive"></script>
-
-		<script id="template" type="text/ractive">
-			<p>{{greeting}}, {{name}}!</p>
-		</script>
-
-		<script>
-			var ractive = new Ractive({
-				target: '#target',
-				template: '#template',
-				data: { greeting: 'Goodbye', name: 'Earth' }
-			});
-		</script>
-	</body>
-</html>
+
+<head>
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.2/css/bootstrap.min.css" integrity="sha384-PsH8R72JQ3SOdhVi3uxftmaW6Vc51MKb0q5P2rRUpPvrszuE4W1povHYgTpBfshb"
+	 crossorigin="anonymous">
+</head>
+
+<body>
+	<div class="container">
+		<div class="row">
+			<div class="col">
+				<div id="target"></div>
+			</div>
+		</div>
+	</div>
+
+	<script src="https://cdn.jsdelivr.net/npm/ractive"></script>
+
+	<script id="template" type="text/ractive">
+		<h1>{{title}}</h1>
+		<p>{{subtitle}}</p>
+		<table class="table">
+			<tr>
+				<th>Name</th>
+				<th>age</th>
+				<th>isMinor</th>
+				<th>Edit</th>
+				<th>Action</th>
+			</tr>				
+		  	{{#each people}}		
+		  	<tr>
+				<td>{{name}}</td>
+				<td>{{age}}</td>
+				<td>{{isMinor}}</td>
```

Comment:
Currently there is a bug that isMinor wouldn't be updated with a change to age.  Consider using {{age < 21}} in the table or changing the edit so that it recalculates the isMinor.